src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -826,7 +826,7 @@ def __init__(self, config):
             ACT2FN["relu"],
```

Comment:
```suggestion
ACT2FN[config.hidden_act],
```
?

Reply:
I can't do this, since `config.hidden_act` is set to "gelu" and used in the backbone